src/iframe/HtmlIframe.jsx

```diff
@@ -52,6 +52,16 @@ export default class HtmlIframe extends React.PureComponent {
     );
   }
 
+  get useNoScript() {
```

Comment:
if props.disableNoScript is available it will must override default settings from $Settings.